cmd/startup/startup.go

```diff
@@ -380,6 +380,13 @@ func startIngestServer(serverAddr string) {
 			}
 		}
 	}()
+
+	go func() {
```

Comment:
we should do this inline and not a goroutine. Because we dont want to start accepting new requests until we have rcovered the wal